rules/community/packetbeat/packetbeat_blacklist_domain.py

```diff
@@ -0,0 +1,17 @@
+"""Alert on PacketBeat events"""
+
+from streamalert.shared.rule import rule
+
+
+DNS_BLACKLIST = [
+    'evil.com.'
+]
+
+@rule(logs=['packetbeat:dns'])
+def packetbeat_blacklisted_domain(rec):
+    """
+    author:       gavin (gavinelder)
+    description:  Lookup for BlackListed DNS (CnC).
```

Comment:
Can you add one more field to include the packetbeat documentation for users who haven't used packetbeat before.
```
reference: https://www.elastic.co/guide/en/beats/packetbeat/master/packetbeat-overview.html
```

Reply:
@chunyong-lin done.